src/components/Controls/Controls.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function Controls({ query, setQuery, handleSubmit }) {
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <input type="text" value={query} onChange={(e) => setQuery(e.target.value)} />
+        <button type="submit" onClick={(e) => setQuery(e.target.value)}>
```

Comment:
Any change to the input above is already setting the query state and triggering a new fetch. This button only sets loading to true. `e.target.value` here is not the correct value either